en/identity-server/7.0.0/docs/deploy/security/keystores/manage-ca-signed-certificates-in-a-keystore.md

```diff
@@ -1 +1 @@
-{% include "../../../../../../includes/guides/encryption/manage-ca-signed-certificates-in-a-keystore.md" %}
+{% include "../../../../../../includes/deploy/security/keystores/index.md" %}
```

Comment:
Is this correct?

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4839